### DISCUSSION TITLE
tabby private key is not work.

### DISCUSSION THREAD

**Discussion by LuckRookie** [Eugeny/tabby#9889] [Q&A] [answered]
The same ssh. Termius and iterm2 can work well, but tabby not work and alerts:All configured authentication methods failed
The important thing is that up until an hour ago, this ssh was still working
![QQ_1724161774193](https://github.com/user-attachments/assets/03d7b35f-7521-438a-a2da-c574b4cef2a8)
notes: The ssh is from ~/.ssh/config file

**Comment by xi67** [comment]
Same here. 

Connection OK vi Windows terminal : ssh user@host. Asking fos passphrase and Connect but not working in Tabby.

**Comment by x3evo** [comment]
same issue. tested with putty works fine but not in tabby

**Comment by Eugeny** [comment, marked as answer]
Check out #9810
